google/compute/compute.yaml

```diff
@@ -0,0 +1,23 @@
+# Google Cloud Pub/Sub API service configuration
+
+type: google.api.Service
+config_version: 3
+name: compute.googleapis.com
+
+title: Google Compute Engine API
+
+documentation:
+  summary:
+    Google Compute Engine Service
+
+apis:
+- name: google.compute.v1.InstancesService
+
+authentication:
+  rules:
+    - selector: '*'
+      oauth:
+        canonical_scopes: |-
```

Comment:
Looks like every method in the Discovery doc has its own scopes. That makes things interesting. As a simple starting point, we could simply union them all to generate this.